README.md

```diff
@@ -133,15 +133,7 @@ Below are some commonly encountered issues and possible ways to resolve them. If
 
    > **On Android,** if you get an error message that includes `INSTALL_FAILED_UPDATE_INCOMPATIBLE` this may be because you previously installed a newer version of the app for your emulator (e.g. on a new branch or testing someone else's pull request) then you switched back to an earlier version. Uninstall the app from your emulator with the command `adb shell pm uninstall org.scottishtecharmy.volunteerapp` then run `npm run android` again.
 
-4. Optional: Update the cached projects data from Jira _(during development, you probably only need to use this if you need the very latest data from Jira or to update project video files whose URLs expire after an hour)_. Open another command terminal window, go to the `api` folder inside the project.
-
-   - If you want to manually update cached projects data, enter this command: `node cron-jobs/run-projects.js`
-     > During development, it's preferable to do this than to run the scheduled cron job described below.
-   - If you want to automatically run all cron jobs regularly , enter this command instead: `node cron-jobs/run-cron-jobs.js` Leave this terminal window open as long as you want this to keep running.
-     > See more about [cron jobs](#cron-jobs) below
-     > Be careful if using this during development: if multiple developers are running this simultaneously, these could conflict if more than one person is updating the same AirTable tables at the same time.
-
-5. Optional: Add event video thumbnail images and update event video files whose URLs expire after an hour. Open another command terminal window, go to the `api` folder inside the project.
+4. Optional: Add event video thumbnail images and update event video files whose URLs expire after an hour. Open another command terminal window, go to the `api` folder inside the project.
```

Comment:
run-cron-jobs.js actually runs both projects and events, so suggest that either remove 4. here and the two bullet points that follow (lines 137-8) -- or in api/server.js require run-projects rather than run-cron-jobs (that way we'd only be running projects cron jobs and you can leave the README here as you've got it now).

The former sets us up ready for events in the future, but I guess the latter (projects only) is all we strictly need for MVP.